manuscripts/report.py

```diff
@@ -633,6 +635,8 @@ def create_pdf(self):
         # Copy the data generated to be used in LaTeX template
         copy_tree(self.data_dir, os.path.join(report_path, "data"))
         copy_tree(self.data_dir, os.path.join(report_path, "figs"))
+        if self.logo:
```

Comment:
You could better do this above, where you define self.logo. That way we have all the functionality in the same place.

Reply:
@jgbarah  it has to be here because of the third line  above this, if a user specifies a new logo ,first the  old logo is copied from templates folder with other data, then the new logo replaces old one.